rust/nft-candy-machine/src/lib.rs

```diff
@@ -447,7 +461,17 @@ pub struct AddConfigLines<'info> {
     #[account(signer)]
     authority: AccountInfo<'info>,
 }
+#[derive(Accounts)]
+pub struct WithdrawFunds<'info> {
+    #[account(
+        mut,
+        has_one = authority
+    )]
+    config: ProgramAccount<'info, Config>,
```

Comment:
In 0.17, you can just use Account, no need for ProgramAccount

Reply:
deal